README.md

```diff
@@ -78,23 +78,17 @@ Example output:
     "id":"36AAB6D3-7DFE-41AB-A0B95A826D4Ctest",
     "previewUrl":"https://d2csxpduxe849s.cloudfront.net/media/test.png",
     "webUrl":"https://d2csxpduxe849s.cloudfront.net/media/test.png",
-    "title":"TestImage"
+    "title":"TestImage",
+    "bynderUrl":"https://test.getbynder.com/media/?mediaId=36AAB6D3-7DFE-41AB-A0B95A826D4Ctest",
+    "updatedAt":"2021-01-20T17:40:14Z",
+    "description" : "test image"
   }
 ]
 ```
 
-## Live site implementation sample
-
-If you want to see live site example of Bynder images displayed on the live site, browse to a [deployed sample site](https://kentico-cloud-sample-app-react-bynder.surge.sh/en-us/articles/cf106f4e-30a4-42ef-b313-b8ea3fd3e5c5).
```

Comment:
If we don't link Bynder anywhere, we can remove the branch from the repository.

Reply:
yes, we can!